test/spec/spec_helpers.js

```diff
@@ -47,6 +47,7 @@ mocha.setup({
 expect = chai.expect;
 
 window.d3 = iD.d3;   // Remove this if we can avoid exporting all of d3.js
+window.sdk = iD.sdk;
```

Comment:
Was reading this issue https://github.com/openstreetmap/iD/issues/4379 and my understanding is that re-export of d3 is done due to d3.event having some internal states => doesn't mean that this should be a common practice for all dependencies.
Curious: Could we just include id-sdk scripts in test/index.html directly (via <script src... )?